tools/data-handler/test/test-data/valid/decision-records/.calc/imports.lp

```diff
@@ -0,0 +1,10 @@
+% ../data-handler/test/test-data/valid/decision-records/.calc/resources
+#include "/home/samu/projects/cyberismo/cyberismo/tools/data-handler/test/test-data/valid/decision-records/.calc/resources/decision/cardtypes/decision-cardtype.lp".
```

Comment:
These absolute paths are updated when running tests, resulting into this file changing (along with modules.lp). Can we use relative paths here?

Reply:
Actually, I think we shouldn't add those to source control

Reply:
I removed them from source control